lib/node_modules/@stdlib/stats/base/snanmeanpn/lib/index.js

```diff
@@ -28,20 +28,17 @@
 * var snanmeanpn = require( '@stdlib/stats/base/snanmeanpn' );
 *
 * var x = new Float32Array( [ 1.0, -2.0, NaN, 2.0 ] );
-* var N = x.length;
 *
-* var v = snanmeanpn( N, x, 1 );
+* var v = snanmeanpn( x.length, x, 1 );
 * // returns ~0.3333
 *
 * @example
 * var Float32Array = require( '@stdlib/array/float32' );
-* var floor = require( '@stdlib/math/base/special/floor' );
 * var snanmeanpn = require( '@stdlib/stats/base/snanmeanpn' );
 *
 * var x = new Float32Array( [ 2.0, 1.0, 2.0, -2.0, -2.0, 2.0, 3.0, 4.0, NaN ] );
-* var N = floor( x.length / 2 );
 *
-* var v = snanmeanpn.ndarray( N, x, 2, 1 );
+* var v = snanmeanpn.ndarray( 4, x, 2, 1 );
```

Comment:
```suggestion
* var v = snanmeanpn.ndarray( 5, x, 2, 1 );
```

The point of the example should be to include a `NaN` value among the iterated elements.